app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+      <%= f.label :photo, 'Profile photo link' %><br />
+      <%= f.text_field :photo, autofocus: true %>
+  </div>
+
+  <div class="field">
+      <%= f.label :bio, 'Bio' %><br />
+      <%= f.text_area :bio, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Kindly design the signup page to look identical to the wireframe provided for this project. 😎 

### Your Design

![image](https://github.com/otmanTR/BlogAppRoR/assets/77106962/d71068b9-3aa3-42ac-b0ad-8a412b668d68)

### Wireframe

![image](https://github.com/otmanTR/BlogAppRoR/assets/77106962/1d737514-6256-4969-b750-9d05220225a9)